src/client/http.ts

```diff
@@ -56,6 +56,8 @@ export type Requester = {
   request: <TResult = unknown>(request: UpstashRequest) => Promise<UpstashResponse<TResult>>;
   requestStream: (request: UpstashRequest) => AsyncIterable<ChatCompletionChunk>;
   headers?: Headers;
+  telemetryHeaders?: Headers;
+  wrapWithGlobalHeaders: (headers: Headers) => Headers;
```

Comment:
previously, `wrapWithGlobalHeaders` was a utility method. Is there a way to keep it that way?

Reply:
I have to pass both fields as arguments so figured it would be easier as a method